db/re/mob_db.txt

```diff
@@ -2664,10 +2664,10 @@
 3502,DR_BASILISK1,Scout Basilisk,Scout Basilisk,20,243,1,95,62,1,11,0,18,10,0,8,17,0,15,0,10,12,1,2,22,0x3885,400,2612,912,288,0,0,0,0,0,0,0,1685,217,1686,217,22985,256,0,0,0,0,0,0,0,0,0,0,0,0,4660,1
 3503,DR_BASILISK2,Combat Basilisk,Combat Basilisk,140,180030,1,583,656,1,1379,0,105,80,56,12,19,11,27,10,10,12,1,2,42,0x3885,250,2456,912,504,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,4661,1
 3504,DR_BASILISK3,Combat Basilisk,Combat Basilisk,148,216036,1,583,656,1,1654,0,126,80,56,12,19,11,27,10,10,12,1,2,42,0x3885,250,2456,912,504,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,4661,1
-3505,DR_BIG_EGGRING,Big Eggring,Big Eggring,25,156,1,97,73,4,82,0,13,2,12,3,5,5,10,5,10,12,1,3,62,0x84,1000,1768,768,576,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,4662,1
+3505,DR_BIG_EGGRING,Big Eggring,Big Eggring,25,142480,0,0,0,1,82,0,63,4,68,12,12,12,68,57,10,12,1,3,62,0x6203695,200,1875,672,480,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,4662,1
```

Comment:
iRO Wiki has more drops for this one. I dont know if we want to take it as a source though.

Reply:
It's fine as it is for now.